lib/utils/find-hosts-host.js

```diff
@@ -8,7 +8,7 @@ module.exports = function findHostsHost() {
   // Keep iterating upward until we don't have a grandparent.
   // Has to do this grandparent check because at some point we hit the project.
   do {
-    if (current.lazyLoading.enabled === true) {
+    if (current && current.lazyLoading && current.lazyLoading.enabled === true) {
```

Comment:
@knownasilya what scenario `current.lazyLoading` is undefined? 

can you add tests for this?